src/jackson.js

```diff
@@ -17,33 +17,51 @@ app.use(express.urlencoded({ extended: true }));
 
 app.get(oauthPath + '/authorize', async (req, res) => {
   try {
-    await oauthController.authorize(req, res);
+    const { redirect_url } = await oauthController.authorize(req.query);
+
+    res.redirect(redirect_url);
   } catch (err) {
-    res.status(500).send(err.message);
+    const { message, statusCode } = err;
```

Comment:
```
    const { message, statusCode = 500 } = err;
```
When error is not JacksonError then we end up sending the right 500 status code instead of undefined.

Reply:
Yes, this is a good catch.